utkdigitalinitiatives/polk-correspondence-app
# Remove forced italics from headnotes on letters.

**JIRA Ticket**: [https://jira.lib.utk.edu/projects/PTC/issues/PTC-75](https://jira.lib.utk.edu/projects/PTC/issues/PTC-75)

# What does this Pull Request do?

- Removes overriding italics on headnotes in letters.
- Allows child elements to decide of italics are required.

# What's new?

Removed `font-style: italic;` from `.tei-postscript` in Sass.
# How should this be tested?

Describe what steps to take to test this change.

- `ant` and upload .xar
- Navigate to Jefferson Davis 4/13/1848
- Review NOTES box and see that only certain child elements have italics and not everything.

# Interested parties
@markpbaggett
